src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.common;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.addressbook.data.person.*;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+public class UtilsTest {
+	
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+	
+	@Test
+	public void isAnyNull_returnsBoolean()
+	{
+		assertTrue(Utils.isAnyNull(Name.EXAMPLE,Phone.EXAMPLE,Email.EXAMPLE,Address.EXAMPLE,null));
+		assertTrue(Utils.isAnyNull(null,Phone.EXAMPLE,Email.EXAMPLE,Address.EXAMPLE,null));
+		assertTrue(Utils.isAnyNull((Object)null));
+		assertTrue(Utils.isAnyNull(new Object[] {null,null}));
+
+		assertFalse(Utils.isAnyNull(Name.EXAMPLE,Phone.EXAMPLE,Email.EXAMPLE,Address.EXAMPLE));
+		assertFalse(Utils.isAnyNull());
+		assertFalse(Utils.isAnyNull(new Object[] {Name.EXAMPLE,Phone.EXAMPLE}));
+		assertFalse(Utils.isAnyNull(new String()));
```

Comment:
Avoid using multiple assert statements. Avoid using assertTrue and assertFalse within the same test case.